roles/kubernetes/node/defaults/main.yml

```
@@ -3,7 +3,7 @@
 kube_apiserver_insecure_bind_address: 127.0.0.1
 
 # advertised host IP for kubelet. This affects network plugin config. Take caution
-kubelet_address: "{{ ip | default(fallback_ips[inventory_hostname]) }}{{ ',' + ip6 if enable_dual_stack_networks and ip6 is defined }}"
+kubelet_address: "{{ ip | default(fallback_ips[inventory_hostname]) }}{{ ',' + ip6 if enable_dual_stack_networks and ip6 is defined else '' }}"
```

```suggestion
kubelet_address: "{{ ip | default(fallback_ips[inventory_hostname]) }}{{ (',' + ip6) if enable_dual_stack_networks and ip6 is defined }}"
```

This is probably closer to what you want as the other would result in a spurious comma at the end.

---

Thanks, but it remains the origin error like:

```yaml
the inline if-expression on line 1 evaluated to false and no else section was defined
```